packages/node/src/integrations/contextlines.ts

```diff
@@ -0,0 +1,139 @@
+import { getCurrentHub } from '@sentry/core';
+import { Event, EventProcessor, Integration } from '@sentry/types';
+import { addContextToFrame } from '@sentry/utils';
+import { readFile } from 'fs';
+import { LRUMap } from 'lru_map';
+
+import { NodeClient } from '../client';
+
+const FILE_CONTENT_CACHE = new LRUMap<string, string | null>(100);
+const DEFAULT_LINES_OF_CONTEXT = 7;
+
+function readTextFileAsync(path: string): Promise<string> {
```

Comment:
Can we leave a todo comment about updating this when we move to the major? (We then will default to Node 8).